src/tm_devices/driver_mixins/device_control/pi_control.py

```diff
@@ -816,20 +842,21 @@ def write(self, command: str, opc: bool = False, verbose: bool = True) -> None:
             Error: An error occurred while sending the command.
             SystemError: ``*OPC?`` did not return "1" after sending the command.
         """
-        if self._verbose and verbose:
-            if "\n" in command:
-                # Format any multiline command to print out with a single timestamp
-                # followed by as many (whitespace padded) f">>  {cmd}" lines as it has
```

Comment:
keep this comment to explain why there's an if check? or now it's not whitespace padded (if i am reading the new code right) so it's not needed?

Reply:
As it stands, it looks like this:
```
[2024-11-14 07:38:35.000785] [tm_devices] [    INFO] (SMU 49 "SMU-DEVICE") Write >>
loadscript loadfuncs
"""Sample script used in test_smu.py."""
print("TEK")
endscript
[2024-11-14 07:39:32.000469] [tm_devices] [    INFO] (SMU 49 "SMU-DEVICE") Write >>  'loadfuncs.save()'
```

I could make changes so that it looks like this:
```
[2024-11-14 07:38:35.000785] [tm_devices] [    INFO] (SMU 49 "SMU-DEVICE") Write >>
    loadscript loadfuncs
    """Sample script used in test_smu.py."""
    print("TEK")
    endscript
[2024-11-14 07:39:32.000469] [tm_devices] [    INFO] (SMU 49 "SMU-DEVICE") Write >>  'loadfuncs.save()'
```

Which do you think is better?

Reply:
I think the single tab looks cleaner, but it shouldn't matter too much if you want to keep it as is. But the docstring comment in your example needs a fix
"""Sample script used in test_smu.py."""
wouldn't be treated as a comment in tsp, use two dashes like:
-- Sample script used in test_smu.py.

Reply:
Ok, I will fix the sample. I will also add the whitespace.

Reply:
Pushed these changes